### PR TITLE
Update host version 3.7.1

### PR DESCRIPTION
**Pull request by v-smanchem**
### Issue describing the changes in this PR

resolves #issue_for_this_pr

### Pull request checklist

* [ ] My changes **do not** require documentation changes
  * [ ] Otherwise: Documentation issue linked to PR
* [ ] My changes **do not** need to be backported to a previous version
  * [ ] Otherwise: Backport tracked by issue/PR #issue_or_pr
* [ ] I have added all required tests (Unit tests, E2E tests)

### CODE REVIEW

File: validateWorkerVersions.ps1 [hunk 1]
```diff
@@ -26,7 +26,7 @@ function removeBomIfExists([string]$data)
     return $data
 }
 
-$cliCsprojPath = "./src/Azure.Functions.Cli/Azure.Functions.Cli.csproj"
+$cliCsprojPath = "C:/Users/v-smanchem/desktop/CoreTools/azure-functions-core-tools/src/Azure.Functions.Cli/Azure.Functions.Cli.csproj"
```
Comment (ejizba) [inline validateWorkerVersions.ps1:29]: Please revert this change

**Review by v-smanchem — COMMENTED** [review]
new changes

File: validateWorkerVersions.ps1 (hunk 1 above)
Comment (v-smanchem) [reply validateWorkerVersions.ps1]: updated
